app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t("sample")
+    if page_title.empty?
```

Comment:
```suggestion
    if page_title.empty? ? base_title : page_title + " | " + base_title
```
Tìm hiểu thêm convention nhé :v